libwds/sink/cap_negotiation_state.cpp

```diff
@@ -102,14 +102,13 @@ std::unique_ptr<Reply> M3Handler::HandleMessage(Message* message) {
           new_prop.reset(new UIBCCapability());
           reply->payload().add_property(new_prop);
       } else if (*it == PropertyName::name[PropertyType::WFD_CONNECTOR_TYPE]){
-          new_prop.reset(new ConnectorType());
+          new_prop.reset(new ConnectorType(0));
```

Comment:
@michaelolbrich, pls update your changes on top of https://github.com/01org/wds/pull/111 when it lands